src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,26 @@
+import { StateContext } from '../../Store';
+import { useContext } from 'react';
+import { TodoItem } from './TodoItem';
+
+export const TodoList = () => {
+  const state = useContext(StateContext);
+  const { todos, filterTodos } = state;
+
+  let list = todos;
+
+  if (filterTodos === 'Active') {
+    list = list.filter(todo => todo.completed === false);
+  }
+
+  if (filterTodos === 'Completed') {
+    list = list.filter(todo => todo.completed === true);
+  }
```

Comment:
better to create some function with filter logic, and use the switch case instead of multiple if